jetty-start/src/main/java/org/eclipse/jetty/start/fileinits/MavenLocalRepoFileInitializer.java

```diff
@@ -56,7 +56,7 @@
         public String version;
         public String type;
         public String classifier;
-        private String mavenBaseUri = "http://central.maven.org/maven2/";
+        private String mavenRepoUri = "https://repo.maven.apache.org/maven2/";
```

Comment:
This should be `https://central.maven.org/maven2/` - Why did you change it to apache.org?

Reply:
I'm simply use to this and it's default one in Apache Maven distribution.

Reply:
I changed it....